index.html

```diff
@@ -0,0 +1,202 @@
+</head>
+<body>
+<header>
+  <h1 class="log"><a href="#">Ndonyi</a></h1>
+  <nav>
+    <div class="bar"></div>
+    <div class="bar"></div>
+    <div class="bar"></div>
+  </nav>
+</header>
+<section class="container">
+<section class="header-text">
+    <h1>I’m Ndonyi <br>
+        Glad to see <br>you!</h1>
+  <p>Hello I’m a software developer!
+     I can help you build a product , feature 
+     or website Look through some of my work and 
+     experience! If you like what you see and have a
+      project you need coded,
+     don’t hestiate to contact me.</p>
+</section>
+
+<section class="contact_list">
+  <h3 >LET’S CONNECT</h3>
+    <ul>
+        <li><a href="https://twitter.com/home"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="https://www.linkedin.com/feed/?trk=homepage-basic_google-one-tap-submit"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+        <li><a href="https://github.com/citec-47"><i class="fa-brands fa-github"></i></a></li>
+        <li><a href="https://www.facebook.com/ndonyi.maurice"></i></a></li>
+     </ul>
+    </section>
+</section>
+<section>
+<div class="project">
+    <div class="project_1st">
+        <div class="four">
+            <img src="./Images/IMAGE5.PNG.png" alt="">
+        </div>
+      <div class="detail">
+        <h2>Tonic</h2>
+        <ul>
+            <li><a href="#">CANOPY</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">2015</a></li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="all">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
+        </div>
+    </div>
+   
+  </div>
+<div class="project1">
+    <div class="project_1st">
+        <div class="one">
+            <img src="./Images/Image_4.png" alt="">
+        </div>
+      <div class="detail">
+        <h2>Multi-Post
+          Stories</h2>
+        <ul>
+            <li><a href="#">CANOPY</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">Back End Dev</a></li>
+            <li><i class="fa-sharp fa-solid fa-circle"></i></li>
+            <li><a href="#">2015</a></li>
+          </ul>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="all">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>JAVASCRIPT</li>
+          </ul>
+          <br>
+        <br>
+          <a href="https://www.facebook.com/ndonyi.maurice" class="resume">See Project</a>
```

Comment:
Please consider adding the functionality of press on the button. So when the button is pressed its style changes as follows. To achieve this you can add the following functionality of code into your CSS file.

.resume:active{
    background-color: #2230D2;
    color: white;
}

![image](https://user-images.githubusercontent.com/94223367/219416172-4958d9ca-1fa4-4068-adbb-ef3657bc7b3f.png)